eox_tagging/api/v1/serializers.py

```diff
@@ -58,7 +59,7 @@ def create(self, validated_data):
         target = validated_data.pop("target_object", None)
 
         if target_type and target_type.lower() in MODELS_WITH_COMPOUND_KEYS:
-            data = self.__convert_compound_keys(target, target_type)
+            data = self.__convert_compound_keys(target, target_type.lower())
```

Comment:
About the `__convert_compound_keys`, the separator has to be exactly `: `  (i.e.1 colon **_and_** 1 space)? Can you make it a bit more forgiving? Ignoring the spaces.